### ISSUE TITLE
Catppuccin does too much escaping

### ISSUE THREAD

**Issue by j-xella** [catppuccin/tmux#281]
### Is there an existing issue outlining your problem?

- [X] I have searched the existing issues and they do not solve my problem.

### Describe your problem.

Before, this used to work:

```sh
set -g @catppuccin_session_text "#{=10:#{s/^(TFS|KB|SUMMIT-)([0-9]*).*/KB-\\2/:session_name}}"
```

Basically, I want to apply some formatting to the session name before showing it.

Unfortunately, it does not work any more. For example, session name `KB1234_xyz` should be displayed as `KB-1234`, but now the result is always `KB-\2`.

Checking `tmux show -A`, it seems that catpuccin sets

```
status-left* "...#{=10:#{s/^(TFS|KB|SUMMIT-)([0-9]*).*/KB-\\\\2/:session_name}}..."
```

in other words, the backslashes are escaped. I don't think the plugin should be doing it. If I want to add a `\` as text, I can escape it myself.

### Paste your configuration.

```sh
set -g @catppuccin_session_text "#{=10:#{s/^(TFS|KB|SUMMIT-)([0-9]*).*/KB-\\2/:session_name}}"
```

### What tmux version are you seeing the issue on?

tmux built from recent commit [651891c3]

**Comment by j-xella** [comment]
The full config:

```sh
set -g @plugin 'catppuccin/tmux'
#set -g @catppuccin_flavour 'mocha' # latte,frappe, macchiato or mocha (default)

set -g @catppuccin_window_left_separator "" # "" "null"
set -g @catppuccin_window_middle_separator "▌" # ":" - was not working in one version of catpuccin
set -g @catppuccin_window_right_separator "" # "null" # remove space between windows
set -g @catppuccin_window_status_enable "yes"
# set -g @catppuccin_window_status_icon_enable "no"
set -g @catppuccin_icon_window_current "null" # Other preference could be "★" , but current window is already color-coded
set -g @catppuccin_icon_window_last "" # "" "-" ; the original dash is too big for my taste
set -g @catppuccin_window_default_text "#{s/^(TFS|KB|SUMMIT-)([0-9]*)/KB-\\2/:window_name}"
set -g @catppuccin_window_current_text "#{s/^(TFS|KB|SUMMIT-)([0-9]*)/KB-\\2/:window_name}"

set -g @catppuccin_status_left_separator "" # "null"
set -g @catppuccin_status_right_separator "null"
set -g @catppuccin_status_fill "all"

set -g @catppuccin_status_modules_left "session"
set -g @catppuccin_session_icon "null"
set -g @catppuccin_session_text "#{=10:#{s/^(TFS|KB|SUMMIT-)([0-9]*).*/KB-\\2/:session_name}}█"

set -g @catppuccin_status_modules_right "user host"
set -g @catppuccin_user_icon "null"
set -g @catppuccin_host_icon "null"
```

I restart tmux, rename the session to KB123_aaa, then get this:

![image](https://github.com/user-attachments/assets/7a33d6f2-edae-44a8-8fe8-251aff9e451e)

The session name in the bottom left corner should be shown as `KB-123`

**Comment by j-xella** [comment]
It seems that the problem (for me, at least), is in `get_tmux_batch_option` function. If I bypass it:

![image](https://github.com/user-attachments/assets/3b7fe06b-677a-42ed-8be3-099342fc9e7b)

Everything works:

![image](https://github.com/user-attachments/assets/a63f913e-08a0-474f-a101-31ccccf3a9a7)

**Comment by vdbe** [comment]
@j-xella 
Can you check if #282 fixes it? Just checking out that PR and running `./catppuccin.tmux` in an tmux session should be enough to see if it fixes it.

**Comment by vdbe** [comment]
Going to revert #240 until I can find a fix for this.

**Comment by j-xella** [comment]
#282 fixes the other issue I reported, #280. But not this issue.

**Comment by vdbe** [comment]
@j-xella Can you test https://github.com/vdbe/catppuccin-tmux?
Merged the fix into main of my fork. Works for me but can never be sure.

Problem was that the -v prints the escape string/value and without it you get the unescaped/value string.
![image](https://github.com/user-attachments/assets/d8b56177-78b0-4e61-b8b6-f056a803b4c7)

**Comment by j-xella** [comment]
Seems to be working. See the screenshot, you can compare the names of sessions/windows with what is displayed in the status bar.

![image](https://github.com/user-attachments/assets/2e8ada35-bfe3-49ab-bac8-95c7a117101d)
